fMRIVolume/GenericfMRIVolumeProcessingPipeline.sh

```diff
@@ -220,6 +220,43 @@ then
 	log_Err_Abort "the --usejacobian option must be 'true' or 'false'"
 fi
 
+
+# ------------------------------------------------------------------------------
+#  Legacy Style Data Options
+# ------------------------------------------------------------------------------
+
+DoSliceTimeCorrection=`opts_GetOpt1 "--doslicetime" $@`              # Whether to do slicetime correction (TRUE), FALSE to omit
+SliceCorrectionDirection=`opts_GetOpt1 "--slicetimedir" $@`          # direction: empty for up, --down for down
+SliceCorrectionInterleaved=`opts_GetOpt1 "--slicetimeodd" $@`        # interleaved: empty for not, --odd for interleaved
+TR=`opts_GetOpt1 "--tr" $@`                                          # TR of the timeseries
+
+# Defaults
+DoSliceTimeCorrection=${DoSliceTimeCorrection:-FALSE}                # No slice timing correction is done by default 
+
+
+# ------------------------------------------------------------------------------
+#  Compliance check
+# ------------------------------------------------------------------------------
+
+MPPMode=`opts_GetOpt1 "--mpp-mode" $@`
+MPPMode=`opts_DefaultOpt $MPPMode "HCPStyleData"`
+Compliance="HCPStyleData"
+ComplianceMsg=""
+
+# -- Slice timing correction
+
+if [ "${DoSliceTimeCorrection}" = 'TRUE' ]; then
```

Comment:
Do we have reason to believe that slice timing correction damages good data?  If not, do we really want to restrict it to legacy data mode?

Reply:
My understanding was that once the TR gets lower than 1s, slice timing correction (and the issue of its interaction with motion correction) can be avoided. My suggestion for this PR would be to support the "simple" slice timing options and limit it to legacy, single band data. I would let @mharms and @glasserm chime in on whether we'd want to extend that to support more complex multi-band options, or rather switch to emerging possibilities of doing simultaneout slice-timing and motion-correction.

Reply:
That was not my question - do we have reason to believe that adding this type of slice timing correction to processing of good data is actually harmful?  If not, then we may not need to categorize it as only allowed for legacy data.  Granted, the slice timing options don't currently support multiband data, and I don't know whether single-band data can meet our guidelines, so maybe it is moot.

Reply:
Slice timing correction as presently implemented is a legacy technique.